package.json

```diff
@@ -17,7 +17,7 @@
     "ci": "yarn test",
     "dev": "DEBUG=info,warn,error babel-node --extensions '.ts,.js' --inspect index.js",
     "dump-schema": "babel-node --extensions '.ts,.js' ./scripts/dump-schema.ts",
-    "dump:local": "yarn dump-schema _schemaV2.graphql & wait",
+    "dump:local": "USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA=false yarn dump-schema _schemaV2.graphql & wait",
```

Comment:
This is b/c we know that the `_schemaV2.graphql` file won't meaningfully change based on this flag being enabled or not. However - without this we might see noisy updates to the file toggle back and forth as people (who may have the flag enabled or not) make PR's to Metaphysics - the updates to the file are more whitespace/auto-generated based on which schema is getting used, and so we can/should suppress that for this purpose.

Reply:
So with `USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA`, we can toggle this to on while developing the schema migration, and then once complete remove all references from the codebase and flatten things out in terms of logic?

Reply:
Yep - at the end when we're all done having an unstitched schema that's based on the flag - I do anticipate a fair number of occurrences of the string `USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA` in the repo. Probably concentrated in several files, but definitely (like we should always be doing!) - we shouldn't enable the flag and walk away - but enable the flag, and within 1-2 sprints after that - come back here and do the cleanup once we know we're good.

I was seeing relatively meaningless updates to the `_schema` file locally based on the flag, so I'm just predicting confusion from wider devs ('wait why did this change, i dont have the flag enabled'), plus this CI script - https://github.com/artsy/metaphysics/blob/6fb7293eff13b17e1472150be114b0c69c899cb3/scripts/ensure-schema-update.sh#L14 - will also confusingly fail back and forth.

Reply:
Nice, and it seems like we can check our work — i.e. verifying that the migration is fully schema-compatible with the stitched version — by dumping the schema with `true` and `false` values for the env var, and confirming that there is no diff in the resulting files.

(With the caveat that this wouldn't actually verify the behavior of any new resolvers, but presumably test coverage comparable to what's in Gravity now would do that.)